pandas/core/generic.py

```diff
@@ -8517,7 +8517,34 @@ def align(
         """
 
         method = missing.clean_fill_method(method)
-
+        """
+        Examples
+        --------
+        >>> df1 = pd.DataFrame([[1,2,3,4], [6,7,8,9]],
+                                 columns=['D', 'B', 'E', 'A'], index=[1,2])
```

Comment:
also, the formatting's a bit strange here, it would be better to either format your example with `black`, or to align the arguments, like
```suggestion
        >>> df1 = pd.DataFrame(
        ...     [[1,2,3,4], [6,7,8,9]],
        ...     columns=['D', 'B', 'E', 'A'],
        ...     index=[1, 2]
        ... )
```

(same for the other parts of the example)